### PR TITLE
wip_lapi: add cscli alert list <alert_id>

### PR DESCRIPTION
**Pull request by AlteredCoder**
- Added new route in LAPI : `GET /v1/alerts/:alert_id`
- Added possibility to specify args for `cscli alerts list <alert_id>` : 

```bash
$ sudo cscli alerts list 193

################################################################################################

 - ID         : 193
 - Date       : 2020-10-21T14:42:16Z
 - Machine    : N/A
 - Simulation : false
 - Reason     : crowdsecurity/ssh-bf
 - Scope:Value: Ip:106.13.112.143
 - Country    : CN
 - AS         : 

 - Active Decisions  :
+-----+-------------------+--------+-----------------------+----------------------+
| ID  |    SCOPE:VALUE    | ACTION |      EXPIRATION       |      CREATED AT      |
+-----+-------------------+--------+-----------------------+----------------------+
| 193 | Ip:106.13.112.143 | ban    | -365h13m23.900360074s | 2020-10-21T14:42:16Z |
+-----+-------------------+--------+-----------------------+----------------------+

 - Events  :

- Date: 2020-10-21 14:42:16 +0000 UTC
+-------------+-----------------+
|     KEY     |      VALUE      |
+-------------+-----------------+
| service     | ssh             |
| log_type    | ssh_failed-auth |
| source_ip   | 106.13.112.143  |
| IsoCode     | CN              |
| IsInEU      | false           |
| ASNNumber   |               0 |
| target_user | Teamspeak       |
+-------------+-----------------+

- Date: 2020-10-21 14:42:16 +0000 UTC
+-------------+-----------------+
|     KEY     |      VALUE      |
+-------------+-----------------+
| IsInEU      | false           |
| ASNNumber   |               0 |
| target_user | Teamspeak       |
| service     | ssh             |
| log_type    | ssh_failed-auth |
| source_ip   | 106.13.112.143  |
| IsoCode     | CN              |
+-------------+-----------------+

- Date: 2020-10-21 14:42:16 +0000 UTC
+-------------+-----------------+
|     KEY     |      VALUE      |
+-------------+-----------------+
| ASNNumber   |               0 |
| target_user | Teamspeak       |
| service     | ssh             |
| log_type    | ssh_failed-auth |
| source_ip   | 106.13.112.143  |
| IsoCode     | CN              |
| IsInEU      | false           |
+-------------+-----------------+

- Date: 2020-10-21 14:42:16 +0000 UTC
+-------------+-----------------+
|     KEY     |      VALUE      |
+-------------+-----------------+
| log_type    | ssh_failed-auth |
| source_ip   | 106.13.112.143  |
| IsoCode     | CN              |
| IsInEU      | false           |
| ASNNumber   |               0 |
| target_user | Teamspeak       |
| service     | ssh             |
+-------------+-----------------+

- Date: 2020-10-21 14:42:16 +0000 UTC
+-----------+-----------------+
|    KEY    |      VALUE      |
+-----------+-----------------+
| service   | ssh             |
| log_type  | ssh_failed-auth |
| source_ip | 106.13.112.143  |
| IsoCode   | CN              |
| IsInEU    | false           |
| ASNNumber |               0 |
+-----------+-----------------+

- Date: 2020-10-21 14:42:16 +0000 UTC
+-----------+-----------------+
|    KEY    |      VALUE      |
+-----------+-----------------+
| source_ip | 106.13.112.143  |
| IsoCode   | CN              |
| IsInEU    | false           |
| ASNNumber |               0 |
| service   | ssh             |
| log_type  | ssh_failed-auth |
+-----------+-----------------+
```

### CODE REVIEW

**Comment by erenJag** [comment]
* Maybe the display of each event can be little noisy in output. 
* Why not having `cscli alerts list -id <ID>` ?

File: cmd/crowdsec-cli/alerts.go [hunk 1]
```diff
@@ -170,13 +236,14 @@ func NewAlertsCmd() *cobra.Command {
 		Use:   "list [filters]",
 		Short: "List alerts",
 		Example: `cscli alerts list
+cscli alerts list <alert_id>
```
Comment (buixor) [inline cmd/crowdsec-cli/alerts.go:239]: this one needs to be removed, moved in a separate command

**Review by buixor — CHANGES_REQUESTED** [review]

File: cmd/crowdsec-cli/alerts.go (hunk 1 above)
Comment (buixor) [reply cmd/crowdsec-cli/alerts.go]: self fix

**Review by buixor — APPROVED** [review]
lgtm

**Pull request merged**